### DISCUSSION TITLE
Any plan to implement mujoco for collision checking?

### DISCUSSION THREAD

**Discussion by msnh2012** [tesseract-robotics/tesseract#1181] [General]

**Comment by Levi-Armstrong** [comment]
I am always interested in including more collision checking capabilities. Do you know of any examples showing how to use mujoco just for collision checking?

**Comment by msnh2012** [comment]
Actually no. I'm learning collision algos and looking for better performance collision engines to use. As far as I know, mujoco has better performance for reinforcement learning. So just wonder whether tesseract has any plan to implement it.
